src/app/Instance/BridgeOverview/Components/BODashboardTableView.tsx

```diff
@@ -47,69 +62,76 @@ export const BODashboardTableView = (
-                              <a href="#">
-                                {" "}
-                                <Truncate
-                                  content={processor.name}
-                                  className={
-                                    "BO-Dashboard__resource__truncated-string"
-                                  }
-                                />
-                              </a>
-                            </Text>
-                          </TextContent>
-                        </FlexItem>
-                        <FlexItem>
-                          {processor.status != ManagedResourceStatus.Ready && (
-                            <SEStatusLabel
-                              status={processor.status}
-                              resourceType={"bridge"}
-                              requestedAt={new Date()}
-                            />
-                          )}
-                        </FlexItem>
-                      </Flex>
-                    </StackItem>
-                    <StackItem
-                      key={rowIndex}
-                      className={"BO-Dashboard__resource__labels"}
-                    >
-                      <LabelGroup>
-                        {processor.connectors?.map((connector, rowIndex) => (
-                          <Label key={rowIndex} color="green">
-                            <Truncate
-                              content={connector}
-                              className={
-                                "BO-Dashboard__resource__truncated-string"
-                              }
-                            />
-                          </Label>
-                        ))}
-                      </LabelGroup>
-                    </StackItem>
-                  </Stack>
-                </Td>
-                <Td isActionCell>
-                  {rowActions ? <ActionsColumn items={rowActions} /> : null}
-                </Td>
-              </Tr>
-            ))}
+          {itemsList?.map((item) => (
+            <Tr key={item.id}>
+              <Td>
+                <Stack>
+                  <StackItem>
+                    <Flex flexWrap={{ default: "nowrap" }}>
+                      <FlexItem>
+                        <TextContent>
+                          <Text component="h4">
+                            <Link to={item.url}>
+                              {" "}
+                              <Truncate
+                                content={item.name}
+                                className={
+                                  "BO-Dashboard__resource__truncated-string"
+                                }
+                              />
+                            </Link>
+                          </Text>
+                        </TextContent>
+                      </FlexItem>
+                      <FlexItem>
+                        {item.status != ManagedResourceStatus.Ready && (
```

Comment:
IIUC we do not want to show the ready status for each processor.

Reply:
Yes. We don't display "Ready" status for the processors.